api/external/ingest/chef.proto

```diff
@@ -65,7 +65,7 @@ service ChefIngester {
     option (chef.automate.api.policy).resource = "ingest:nodes";
     option (chef.automate.api.policy).action = "delete";
     option (chef.automate.api.iam.policy).resource = "infra:nodes";
-    option (chef.automate.api.iam.policy).action = "infra:ingestNodes:delete";
+    option (chef.automate.api.iam.policy).action = "infra:ingest:delete";
```

Comment:
I noticed we didn't add the action `infra:ingestNodes:*` in any roles resulting in some unexpected failures when testing `/ingest/events/chef/node-multiple-deletes`. 
I decided to simplify the action name instead of adding it to roles, but I'm open to discussing that choice.